photoeditor/src/main/java/ja/burhanrashid52/photoeditor/PhotoEditor.kt

```diff
@@ -193,6 +193,13 @@ interface PhotoEditor {
      */
     fun redo(): Boolean
 
+    /**
+     * How many redo operations are available.
+     *
+     * @return size of redo stack
+     */
+    val redoStackCount: Int
```

Comment:
Rather than expose a count, we should expose a boolean to allow to check redo is available or not.
```
val isRedoAvailable: Boolean
val isUndoAvailable: Boolean
```
And will expose both operations.

Reply:
Someone might want to show for example badge with number of available undo/redo operations. So I kept `redoStackCount`, added `undoStackCount` and also added helper functions `isUndoAvailable` and `isRedoAvailable`. Is this solution will be OK?

Reply:
Agreed that it would be an additional functionality; however, I am more inclined towards "introducing a feature when enough people ask for it." As far as I remember, I don't see any issue or PR asking for it. Also, I have not come across an app where they show undo/redo badge count.

Reply:
Done